.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,46 @@
+name: Update IntelliJ Version
+
+on:
+  pull_request:
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          if [[ -n $(git status | grep 'modified') ]]
```

Comment:
https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/.circleci/config.yml#L35

1. `git status` is a [**porcelain** command](https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain). While it's probably correct what you do here, it's still preferred to use **plumbing** commands in automated setups like CI or scripts. E.g. `git machete` doesn't use the porcelain `git branch` under the hood, but plumbing/lower-level `git for-each-ref` instead.
2. Subjective, but I prefer doing `if ! ...; then exit; fi; .........` than `if ...; then .........; fi`; handling indent levels is tricky enough in YAML, even without language injections